### ISSUE TITLE
Can you share how you generated 'frac_vecs' ?

### ISSUE THREAD

**Issue by taehwakkwon** [ajtejankar/mixtral-vis-moe#1]
Hello. Thank you for sharing your great visualization code. It is amazing! 

I am trying to visualize other MoE model with your code. 
So, I get mmlu data and do forward to get routers logits with Mixtral-8x7B-Instruct-v0.1. After getting topk=2 indices, I visualized the results using your code.  However, the result is not similar to yours. The way I obtained frac_vecs is described below:

1. I tokenized 'cais/mmlu' with apply_chat_template using a batch size of 1.
2. I performed a forward pass and obtained the topk=2 indices.
3. I summed through the tokens and checked which experts were selected. The shape is (layer, experts).
4. I normalized it by the number of tokens."

Can you share your code to generate frac_vecs or tell me what was wrong with mine?
